src/undo.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+const readline = require('readline');
+const tty = require('tty');
+const chalk = require('chalk');
+
+const {
+  getConfig,
+  logUndoConfirmation,
+  logUndoConclusion
+} = require('./helpers');
+const {
+  readDirPromise,
+  rimrafPromise,
+  statPromise
+} = require('./utils');
+
+const config = getConfig();
+
+// Delete component directory
+readDirPromise(config.dir).then(files => {
+  if (files.length === 0) {
+    console.log(`There\'s no files in ${config.dir}! Is this where your components
+      are located?`);
+    process.exit(0);
+  }
+
+  let lastCreatedFile = { name: '', dateModified: 0 };
+
+  // Create promises to check which component was created last
+  const filePromises = files.map(name => {
+    return statPromise(`${config.dir}/${name}`).then(fileStat => {
+      const dateModified = fileStat.mtime.getTime();
+      if (lastCreatedFile.dateModified < dateModified) {
+        lastCreatedFile = { name, dateModified };
+      }
+    });
+  });
+
+  // Invoke promises and get last modified component
+  Promise.all(filePromises).then(() => {
+    const componentName = lastCreatedFile.name;
+    // Log the intro
+    logUndoConfirmation(componentName);
```

Comment:
Hm... What do you think about doing a check, to see if the component was made in the last 60 seconds, and skipping the confirmation if it was?

The times that i've wanted an "undo", I realize almost instantly, and I haven't even touched the generated files yet. So I think it would be nice to skip the confirmation if less than a minute has passed.

This would require a bit of a rewrite, to separate the "asking" from the "deleting", but I think this is a good refactor to do anyway :)